### ISSUE TITLE
Crysis next perhaps...?

### ISSUE THREAD

**Issue by keybreak** [CookiePLMonster/SilentPatchFarCry#3]
1. Huge fan of your work, especially on GTA!!

2. There's only one real issue i ever had in Crysis...Guess what, freaking censorship!
In 1st version of game's .exe there were full body physics available (you can move bodies by shooting, vehicles etc etc), later with patches (which are really necessary it seems, since they fix a lot of stuff) they removed it due to some goddamn censorship and german laws! Although physics itself of course it still there, you can easily see it by shooting some explosive barrel near dead bodies...

Seems they removed some parts of code which affect any actions specifically between player and dead body..
More info on that https://gamefaqs.gamespot.com/boards/931665-crysis/52605922

I feel like you're the only man alive up for challenge :+1: 
It hugely affects realism of one of best games ever made..

**Comment by mirh** [comment]
~~Mhh.. seems like it's [true](https://web.archive.org/web/20080202025552/http://crymod.com/thread.php?threadid=12433) they had it blocked everywhere.~~ Or it could have been just due to where crytek was hosting their servers, rather than selling their games..
EDIT2: [yup](http://web.archive.org/web/20140729100948/http://www.crydev.net/viewtopic.php?f=277&t=44321)
There is [already](http://web.archive.org/web/20071128140401/http://www.incrysis.com/index.php?option=com_content&task=view&id=571) a mod for ragdoll though?

**Comment by keybreak** [comment]
@mirh 
Yep, but i was unable to make it actually work, you had better luck?
Also it doesn't work with v1.2 which should be latest, not sure about steam though

**Comment by keybreak** [comment]
Oh, also i remembered some stuff like Crysis's .exe have some artifitial engine limitations on one of parameters (max number of tris or max number of fully rendered tree nodes with max lods, something like that - i'll try to remember once install it again) anyway, if you reach certain limit in config it either set exposure of scene to almost full white or just crash.

But, thing is that this problem of limitations was fixed in Crysis Warhead, some people just use Warhead's exe...But that brings some problems too, so that's also probably something interesting to figure out :+1: 

Other than that...Crysis's config is so supreme to any other game, i can't remember anything deal breaking like that.

**Comment by mirh** [comment]
Mhh so.. I much very *not* test games to be fair. 
Anyway, I seem to have find that [first](https://www.playground.ru/files/crysis_ragdoll_blood_mod-9529/) you could just edit a stupid lua file and 3 xml, and call it a day. 
[Then](https://web.archive.org/web/20090207165652/http://www.nvnews.net/vbulletin/showthread.php?t=105854) it seems like you also needed some dlls, and last but not least.. Idk it just seems the makers at GSC/Spielegeier.de did make an [installer](https://down.gamersky.com/pc/200805/13838.shtml) (which may not have been the only existing [fix](https://gamefaqs.gamespot.com/boards/931665-crysis/42543960) but whatever, it's the only one that you can find for download). 
[Allegedly](https://www.forum-3dcenter.org/vbulletin/showthread.php?t=429628) we are just talking about downgrading the dlls with those of the vanilla release though.

**Comment by keybreak** [comment]
No it seems to be .exe, with any exe other than v1.0 no go

**Comment by mirh** [comment]
The launcher remake guy has this almost completely handled. So we can close I think. 
If any, I'm now [getting word](https://github.com/kcat/dsoal/issues/33#issuecomment-1001251683) FC1 seems to be fucked up badly as far as sound goes.. And you'd wonder if it couldn't be checked.

**Comment by keybreak** [comment]
@mirh 
Have there been any changes since last time i've tested it there?
It wasn't ideal...
